src/constants.js

```diff
@@ -41,3 +41,11 @@ export const COUNTRIES = [
     svgActive: tanzaniaActive
   }
 ];
+
+export const COUNTRIES_DEFAULT_VIEWPORTS = {
+  CAN: { zoom: 2, latitude: 56.1351095, longitude: -106.3460756 },
+  SWE: { zoom: 4, latitude: 62.295911, longitude: 17.861953 },
+  IDN: { zoom: 3, latitude: -4.341701, longitude: 122.389996 },
+  PER: { zoom: 4, latitude: -10.471441, longitude: -75.135125 },
+  TZA: { zoom: 5, latitude: -5.790175, longitude: 36.718777 }
+};
```

Comment:
What about using bounds prop instead of viewport?

Reply:
List of countries bounding boxes: https://gist.github.com/graydon/11198540

Reply:
Like here? https://github.com/Vizzuality/gfw/blob/develop/app/javascript/components/ui/map/component.jsx#L167

Reply:
Like here: https://github.com/Vizzuality/gfw/blob/develop/app/javascript/components/ui/map/component.jsx#L105

Reply:
Good point, it seems to be a much more elegant solution, but I'd skip it for now, considering that we want to deliver working carto layers on Monday. But it should go to our tasks list for sure :+1:

Reply:
should we merge then, and keep that in the task list?

Reply:
I'd vote for that